Makefile

```diff
@@ -1,9 +1,9 @@
 #############################################################################
```

Comment:
The `Makefile` is not needed in the repo. It adds complexity since every user/PR is doing a lot of useless changes mostly affected by the local structure of each system.

Reply:
I deleted it.